scheme/sevsnp/evidence_handler_test.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package sevsnp
+
+import (
+	"encoding/json"
+	"github.com/stretchr/testify/assert"
+	"github.com/veraison/ear"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/veraison/services/proto"
+)
+
+var testNonce = []byte{
+	0x07, 0x06, 0x05, 0x04, 0x03, 0x02, 0x01, 0x00,
+	0x0f, 0x0e, 0x0d, 0x0c, 0x0b, 0x0a, 0x09, 0x08,
+	0x17, 0x16, 0x15, 0x14, 0x13, 0x12, 0x11, 0x10,
+	0x1f, 0x1e, 0x1d, 0x1c, 0x1b, 0x1a, 0x19, 0x18,
+}
+
+func Test_ExtractClaims_ok(t *testing.T) {
+	tokenBytes, err := os.ReadFile("test/sevsnp-ratsd-token")
+	require.NoError(t, err)
+
+	taEndValBytes, err := os.ReadFile("test/ta-endorsement.json")
+	require.NoError(t, err)
+
+	handler := &EvidenceHandler{}
+
+	token := proto.AttestationToken{
+		TenantId:  "0",
+		Data:      tokenBytes,
+		MediaType: EvidenceMediaTypeRATSd,
+		Nonce:     testNonce,
+	}
+	ta := string(taEndValBytes)
+	_, err = handler.ExtractClaims(&token, []string{ta})
+
+	require.NoError(t, err)
+}
+
+func Test_ValidateEvidenceIntegrity_ok(t *testing.T) {
```

Comment:
Also, I saw three types of Evidence Format in the code, but I see ONLY tests with RATSD Token only,

Can you please add the test cases for TSM Report in JSON and CBOR as well ?